modules/apps/object/object-api/src/main/java/com/liferay/object/deployer/ObjectDefinitionDeployer.java

```diff
@@ -20,7 +24,39 @@ public interface ObjectDefinitionDeployer {
 	public List<ServiceRegistration<?>> deploy(
 		ObjectDefinition objectDefinition);
 
+	public default Map<Long, List<ServiceRegistration<?>>>
+		deployObjectDefinitions(
+			long companyId,
+			ObjectDefinitionLocalService objectDefinitionLocalService) {
```

Comment:
@pedro-oliveira446 It's better passing the objectDefinitions instead of objectDefinitionLocalService. Is there any restriction about it?

Reply:
Done!